ecosystem-overview.md

```diff
@@ -4,7 +4,7 @@
 One of nGraph’s key features is framework neutrality. We currently support 
 popular deep learning frameworks such as TensorFlow and MXNet with stable 
 bridges to pass computational graphs to nGraph. Additionally nGraph 
-Compiler has functional bridges to PaddlePaddle and PyTorch (via [ONNXIFI]). 
```

Comment:
@indie Since PyTorch and ONNXIFI are no longer supported by nGraph, I've changed this sentence.